samples/net/sockets/http_server/src/headers/server_functions.h

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
This likely needs to be moved along with `server.c` to `subsys/net/lib/http/`

To be clear, private API functions should most likely be declared separately from public API functions, and ideally private API functions would just be declared static in `server.c`.

Please move this file to `subsys/net/lib/http/http_server_functions.h`. If these are not supposed to be publicly available functions, the please declare them as st